recipe/meta.yaml

```diff
@@ -26,17 +26,25 @@ requirements:
     - python
     - pip
     - setuptools
+    {% if not compiler('c').startswith("toolchain") %}
+    - llvm-openmp  # [osx]
+    {% else %}
+    - openmp 4.0.*  # [osx]
     - clangdev 4.0*  # [osx]
     - libcxx 4.0.*  # [osx]
-    - openmp 4.0.*  # [osx]
+    {% endif %}
   run:
     - python
     - numpy
     - scipy
     - scikit-learn
+    {% if not compiler('c').startswith("toolchain") %}
+    - llvm-openmp  # [osx]
+    {% else %}
+    - openmp 4.0.*  # [osx]
     - clangdev 4.0*  # [osx]
     - libcxx 4.0.*  # [osx]
-    - openmp 4.0.*  # [osx]
```

Comment:
It might just be worth dropping this legacy compiler case altogether and restricting this to build only with the new compilers on macOS. The OpenMP story was not very good for macOS before and trying to make things work with that old strategy is probably too much effort. Especially since the compiler migration is winding down.